### PR TITLE
[13.0] [MIG] fieldservice_stock

### PR DESCRIPTION
**Pull request by RLeeGML**
Migration of fieldservice_stock to 13.0

#354 

Moved from #651

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [flake8]
-max-line-length = 88
+max-line-length = 80
```
Comment (dreispt) [inline .flake8]: This should stay at 88

File: fieldservice_stock/models/res_territory.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 from odoo import fields, models
 
 
-class FSMTerritory(models.Model):
-    _inherit = "fsm.territory"
+class ResTerritory(models.Model):
+    _inherit = "res.territory"
```
Comment (dreispt) [inline fieldservice_stock/models/res_territory.py:7]: I miss the migration scripts for this

File: fieldservice_stock/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    "version": "13.0.1.0.0",
+    "category": "Field Service",
+    "author": "Open Source Integrators, "
+    "Brian McMaster, "
+    "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/field-service",
+    "depends": [
+        "fieldservice",
+        "stock_request_direction",
+        "stock_request_picking_type",
```
Comment (baimont) [inline fieldservice_stock/__manifest__.py:16]: missing stock_request_submit in order to do https://github.com/OCA/field-service/pull/671/files?file-filters%5B%5D=.py#diff-f7551e26fc5881da48b15fe64ef2878a3d37ea4fae84aeaa5468416742861325R73

**Comment by patrickrwilson** [comment]
@RLeeOSI In v12, PR #675 simplified the fieldservice_stock module. I think we should migrate based on the simplified version since these are still in progress and haven't been merged yet.

**Comment by brian10048** [comment]
@RLeeOSI let me know if you do not have time to do the migration based on changes in #675 and I'll be glad to work on it

**Comment by RLeeGML** [comment]
@brian10048  Since I'm not familiar with the split into fieldservice_stock and fieldservice_equipment_stock/fieldservice_stock_request, I would appreciate if you could handle this. I agree that this solves several issues with dependencies. Let me know if you want any help with migration scripts or unit tests. Are you going to open migration PRs for the new modules?

**Comment by brian10048** [comment]
Closing as this was replaced with #797 
Thank you for the work @RLeeOSI 

For the new modules needing migrated, I don't have much time to work on modules related to stock_request.  Generally speaking, the code is mostly the same for these new modules, just moved from where it was.  A lot of the migration work you did here will apply to the migration of the new modules

**Pull request closed without merging**